packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -660,6 +660,9 @@ Map {
         "isRequired": true,
         "type": "func",
       },
+      "requiresExplicitSelection": Object {
```

Comment:
Would it help if we named this something like `focusMode`? Could be `PropTypes.oneOf(['automatic', 'manual'])`

Reply:
oh yeah I like that!

Reply:
Good suggestion - Used `selectionMode` to better reflect what the new user interaction pattern does.